mobify/progressive-web-scaffold
# Astro: Add beta icon to debug and previewEnabledRelease builds

Add the beta icon to the project, set it as the default for DEBUG and PREVIEWENABLEDRELEASE configurations.

## Changes
- Add icons
- Change Debug configuration to use beta icon

## How to test-drive this PR
- Run app on debug, see beta icon
- Run app on release, see non-beta icon
